Tests/ObservableSpec+From.swift

```diff
@@ -54,11 +54,15 @@ class ObservableFromSpec: QuickSpec {
             context("when there are two .Next() events with content") {
                 beforeEach {
                     let ob = testScheduler.createColdObservable(self.createRecords(
-                        (100, .Next([1,2,3,4])),
-                        (300, .Next([5,6,7])),
-                        (350, .Completed)))
+                        records: (100, .next([1,2,3,4])),
+                        (300, .next([5,6,7])),
+                        (350, .completed)))
                     
-                    observable = testScheduler.scheduleObservable { return Observable.from(ob.asObservable()) }
+                    observable = testScheduler.scheduleObservable {
+                        return ob.flatMap({ (array) -> Observable<Int> in
+                            return Observable.from(array)
+                        })
+                    }
```

Comment:
Could be written in shorter way (as described above). Applies to other similar code.